Unity-Technologies/ScriptableRenderPipeline
# Move LWRP tests out of package and into project (2019.1)

### Purpose of this PR
Moving LWRP tests from com.unity.testing.srp.lightweight to the project in TestProjects to make the tests consistent with other tests in the repo.

---
### Release Notes
Moving LWRP tests from com.unity.testing.srp.lightweight to the project in TestProjects to make the tests consistent with other tests in the repo.

---
### Testing status
**Katana Tests**: 
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?automation-tools_branch=master&ScriptableRenderLoop_branch=move-LW-tests-into-project-2019.1&unity_branch=2019.1%2Fstaging
Failures are from known issues

**Manual Tests**: What did you do?
The tests have been run locally to confirm that they still appear and run in the Test Runner window

**Automated Tests**: What did you setup?
No tests added or removed - they were moved from one place to another.

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
None

**Halo Effect**: None, Low, Medium, High?
None

---
### Comments to reviewers
Apologies that this isn't a proper backport - I actually had this branch set up before I started work on the master PR
